app/Services/Backup/PgBackrestService.php

```
@@ -0,0 +1,364 @@
+
+        if ($targetTime) {
+            $escapedTargetTime = escapeshellarg($targetTime);
+            $cmd .= " --type=time --target={$escapedTargetTime} --target-action=promote";
+        }
+
+        $cmd .= ' restore';
+
+        return $cmd;
+    }
+
+    public static function buildInfoCommand(string $stanza, bool $json = true, ?int $repoNumber = null): string
```

PHPMD (2.15.0)

The method buildInfoCommand has a boolean flag argument $json, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)